sdk/nodejs/Makefile

```diff
@@ -17,10 +17,13 @@ include ../../build/common.mk
 
 export PATH:=$(shell yarn bin 2>/dev/null):$(PATH)
 
-lint::
+ensure::
+	yarn
+
+lint:: ensure
```

Comment:
Same as with Python PR, I wonder why we did not link targets before, I wonder if people have opinions on this. Personally sounds like a change that's worth taking.